db/migrate/20220805161029_devise_create_users.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class DeviseCreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ''
+      t.string :encrypted_password, null: false, default: ''
+
+      ## Recoverable
+      t.string   :reset_password_token
+      t.datetime :reset_password_sent_at
+
+      ## Rememberable
+      t.datetime :remember_created_at
+
+      t.string :name, limit: 50, null: false
+      t.string :username, limit: 50, null: false
+      t.integer :user_type, null: false
+
+      t.belongs_to :store, null: true
```

Comment:
```suggestion
      t.belongs_to :store, null: true, foreign_key: true
```

isso foi algo que deixei passar pru store tambem, se nao passar o `foreign_key` nao vai criar a FK, tem q criar a FK para o stores tambem